internal/ui/diff/diff.go

```diff
@@ -1,21 +1,39 @@
 package diff
 
 import (
+	"github.com/idursun/jjui/internal/config"
 	"github.com/idursun/jjui/internal/ui/common"
+	"github.com/idursun/jjui/internal/ui/context"
 
+	"github.com/charmbracelet/bubbles/key"
 	"github.com/charmbracelet/bubbles/viewport"
 	tea "github.com/charmbracelet/bubbletea"
 )
 
 type Model struct {
-	view viewport.Model
+	view   viewport.Model
+	keymap config.KeyMappings[key.Binding]
+}
+
+func (m *Model) ShortHelp() []key.Binding {
+	// TODO: not sure how to get the full list of mappings that are passed through
+	// to the viewport (eg d scrolls down, u scrolls up)
```

Comment:
They should be available through `viewport`s keybindings. We use the defaults here.

Reply:
something like the below:

```go
km := m.view.KeyMap
return []key.Binding{km.Up, km.Down, km.HalfPageDown, km.HalfPageUp, km.PageDown, km.PageUp, m.keymap.Cancel}
```